platforms/hyperledger-fabric/configuration/roles/create/crypto/orderer/tasks/orderer.yaml

```diff
@@ -40,6 +40,18 @@
     loop_var: files
   when: vault_msp_result.failed == True
 
+############################################################################################
+# This task changes the permissions for msp
+- name: Convert the file to unix format
+  shell: |
```

Comment:
use file module https://docs.ansible.com/ansible/latest/modules/file_module.html instead of shell